src/nvim/log.c

```diff
@@ -14,8 +14,6 @@
 # include <unistd.h>
 #endif
 
-#define USR_LOG_FILE "$HOME/.nvimlog"
```

Comment:
How about just using `_PATHSEPSTR` here instead of moving it to globals.h? (We should probably change the name for `_PATHSEPSTR`, but don't worry about it for now)